locale/en-us/skill.json

```diff
@@ -0,0 +1,30 @@
+{
+    "skill_id": "skill-ovos-hello-world.OpenVoiceOS",
+    "source": "https://github.com/OpenVoiceOS/skill-ovos-hello-world",
+    "package_name": "skill-ovos-hello-world",
+    "pip_spec": "git+https://github.com/OpenVoiceOS/skill-ovos-hello-world@dev",
+    "license": "Apache-2.0",
+    "author": "OpenVoiceOS",
+    "extra_plugins": {
+        "core": ["ovos-utterance-transformer-XXX"],
+        "PHAL": ["ovos-PHAL-XXX"],
+        "listener": ["ovos-audio-transformer-XXX", "ovos-ww-plugin-XXX", "ovos-vad-plugin-XXX", "ovos-stt-plugin-XXX"],
+        "audio": ["ovos-dialog-transformer-XXX","ovos-tts-transformer-XXX", "ovos-tts-plugin-XXX"],
+        "media": ["ovos-ocp-XXX", "ovos-media-XXX"],
+        "gui": ["ovos-gui-extension-XXX"]
```

Comment:
please remove these, should be empty lists, those were just example placeholders